### PR TITLE
Add Yt::Comment resource

### PR DESCRIPTION
**Pull request by MoTai**
Please review :wink:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,23 @@ comment_thread.can_reply? #=> true
 comment_thread.public? #=> true
 ```
 
+Yt::CommentThread
```
Comment (claudiofullscreen) [inline README.md]: Yt::Comment

File: README.md [hunk 2]
```diff
@@ -132,6 +132,23 @@ comment_thread.can_reply? #=> true
...
+
+* Get details of a comment.
+
+```ruby
+Yt::Comment.new id: 'z13vsnnbwtv4sbnug232erczcmi3wzaug'
+
+comment.text_display #=> "awesome"
+comment.author_display_name #=> "Jack"
+comment.like_count #=> 1
+comment.updated_at #=> 2016-3-23
```
Comment (claudiofullscreen) [inline README.md]: is it just a date? or a full timestamp?

File: spec/models/comment_spec.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  describe 'attributes' do
+    examples = {
+      video_id: {with: 'xyz123', without: nil},
+      parent_id: {with: 'abc123', without: nil},
+      text_display: {with: 'awesome', without: nil},
+      author_display_name: {with: 'John', without: nil},
+      like_count: {with: 10, without: nil},
+      updated_at: {input: '2016-03-22T12:56:56.3Z', with: Time.parse('2016-03-22T12:56:56.3Z'), without: nil},
+    }
```
Comment (claudiofullscreen) [inline spec/models/comment_spec.rb:22]: lol so meta

Comment (MoTai) [inline spec/models/comment_spec.rb:22]: Save a lot typing in this case :smile:

**Comment by claudiofullscreen** [comment]
Super cool! Just check the few comments above, then I'll merger!!

Bravo! :clap: :clap: :clap: :clap:

File: README.md (hunk 2 above)
Comment (MoTai) [inline README.md]: It is full timestamp, corrected.

**Comment by MoTai** [comment]
Corrected the errors :smile_cat:

**Comment by claudiofullscreen** [comment]
YEA YEA YEA!!!!

On Thu, Mar 24, 2016 at 4:14 PM, Qiang Han notifications@github.com wrote:

> Corrected the errors [image: :smile_cat:]
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> https://github.com/Fullscreen/yt/pull/270#issuecomment-201068799

## 

 Claudio Baccigalupo  | Manager, Software Engineering  |  Fullscreen, Inc.
http://www.fullscreen.com

**Pull request merged**